tests/integration/rules/length.phpt

```diff
@@ -24,8 +25,21 @@ try {
     echo $e->getMessage().PHP_EOL;
 }
 
+try {
+    v::length(5, 5)->check('123456');
+} catch (LengthException $e) {
+    echo $e->getMessage().PHP_EOL;
+}
+
+try {
+    v::length(3, 5)->assert('phpsp.org.br');
+} catch (AllOfException $e) {
+    echo $e->getFullMessage();
+}
 ?>
 --EXPECTF--
 "nawarian" must have a length lower than 5
 "phpsp.org.br" must have a length greater than 13
-"phpsp.org.br" must not have a length between 5 and 20
\ No newline at end of file
+"phpsp.org.br" must not have a length between 5 and 20
```

Comment:
The exception has 6 possible messages, we should test all of them with `check()` and `assert()`.